### PR TITLE
Do not build .sh scripts from .sh.in

### PR DESCRIPTION
**Pull request by paper42**
The build process only replaces @@MKLIVE_VERSION@@ with the correct version string. This can also be done by simply sourcing a version.sh file. This way it's simpler, avoids one unnecessary build step and people won't accidentally run an old version of a script.

depends on #300

### CODE REVIEW

File: installer.sh [hunk 1]
```diff
@@ -24,6 +24,9 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #-
+#
```
Comment (classabbyamp) [inline installer.sh:27]:
```suggestion
```
don't think this extra comment line is needed

File: version.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+MKLIVE_VERSION="0.23 $(git rev-parse --short HEAD)"
```
Comment (classabbyamp) [inline version.sh:1]: i think having the rev-parse happen at runtime will not work in installer.sh when it's included in a live image

Comment (classabbyamp) [reply version.sh:1]: maybe there could still be a sed to replace the version in the installer done by make-x86-images

Comment (paper42) [reply version.sh:1]: good catch, fixed, this now depends on #300

File: installer.sh (hunk 1 above)
Comment (paper42) [reply installer.sh]: that was a mistake, fixed

**Pull request merged**